app/services/pii/classifier.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Pii
+  class Classifier
+    # @param [Pii::Attributes] pii
+    # @param [Proc] block A proc operate on pii
+    # @return [Boolean|Object]
+    def self.classify(user_info, &block)
+      block.call(user_info)
+    end
```

Comment:
this method looks unused? do we need it?

Reply:
nope, we can remove it.